### DISCUSSION TITLE
Next Major Java SDK Version

### DISCUSSION THREAD

**Discussion by saralvasquez** [braintree/braintree_java#124] [Ideas]
## 👋  Welcome
We're using GitHub Discussions as a place to engage with the members of our community, to share ideas, and to support one another.

## 💡  Discussion Topic: Next Major Version of the Braintree Java SDK
The Braintree SDK team is deciding what to include in our next major version release (v4). We are looking for some insight on:
- Language-specific improvements that you would like to see
- Features you'd like us to consider
- Pain points you'd like to see resolutions to

## 🗣 Let Us Know
Please use this thread to voice your opinions, concerns, or thoughts! We'd love to know how we can make the next version better for you.

**Comment by zericardodarce** [comment]
Hello,
I would like to know if Braintree is planning to support Java 17 in a future release. And if so, when you plan to have it.
Thank you,
Ricardo
